src/Elastic.CommonSchema.Generator/Schema/DTO/Field.cs

```diff
@@ -162,5 +151,18 @@ public class Field
 		/// </summary>
 		[JsonProperty("norms")]
 		public bool? Norms { get; set; }
+
+		/// <summary>
+		///  If <see cref="Type"/> is set to <see cref="FieldType.ScaledFloat"/> this configures the scaling factor
+		/// </summary>
+		[JsonProperty("scaling_factor")]
+		public int? ScalingFactor { get; set; }
+
+		/// <summary>
+		///  Validation (regex) pattern
+		/// </summary>
+		[JsonProperty("pattern")]
+		public string Pattern { get; set; }
```

Comment:
It seems there is a typo in the spec.

`client.mac` has "patther" instead of "pattern" [here](https://github.com/elastic/ecs/blob/main/generated/ecs/ecs_nested.yml#L484). From what I understand currently has 2 effects:
- We loose the defined pattern for the `client.mac` field
- The generator prints a warning

But the generator still works, so it's good.

Nevertheless, I think we should do something here.

Options I see:
- we open a PR to fix it, and the current version is just what it is and we accept it (I think this would be the best)
- we try to workaround and also parse "patthern" (I'd rather not...).

Reply:
++ already addressed this separately: 

https://github.com/elastic/ecs/pull/2014

Having the specification validation really paid off!